profitbricks/src/main/java/org/jclouds/profitbricks/binder/BaseProfitBricksRequestBinder.java

```diff
@@ -56,6 +56,8 @@ protected BaseProfitBricksRequestBinder(String paramName) {
    protected abstract String createPayload(T payload);
 
    protected String formatIfNotEmpty(String pattern, Object param) {
+      if (param == null )
+         return "";
       return Strings.isNullOrEmpty(param.toString()) ? "" : String.format(pattern, param);
```

Comment:
I can't exactly use it, since the param expects a String. (Doing <code>#toString()</code> won't do on null, cast to String throws a <code>ClassCastException</code> for <code>Boolean</code>) Anyway, see the change I did instead